Added Teampopcorn's first blog post and photos

sleepypioneer opened the pull request.

sleepypioneer committed:
added Teampopcorn first blog post and photos

sleepypioneer committed:
updated Markus profile entry

sleepypioneer commented:
@anasofiapinho I have failed checks for the deploy preview. Its because I dont have ruby set up on my home machine, is it ok without this check? I will set it up now incase it's not or can commit again from the coaching company in two hours.

sleepypioneer committed:
moved misty copeland quote to higher up in text

sleepypioneer committed:
Moved title Position

v-thakkar commented on blog/_posts/2018-07-09-welcome-to-team-popcorn.md:
```diff
@@ -0,0 +1,80 @@
+title: Meet Team Popcorn
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/2018-07-09-welcome-to-team-popcorn.md
```
Hi @sleepypioneer! Thanks for the PR. I just have minor comments. You don't need markdown file extension (.md) for 'permalink'.

v-thakkar commented on blog/_posts/2018-07-09-welcome-to-team-popcorn.md:
```diff
@@ -0,0 +1,80 @@
+current: blog
+author: Arati and Jessica
+twitter: TeamPopcornBer
+categories: student-posts
```
Just to maintain consistency across files and format for the posts, can you just change how you have declared categories parameter like below:

```
categories:
- student-posts
```

sleepypioneer committed:
made changes as per request

- don't need markdown file extension (.md) for 'permalink'
- changed format for categories section

sleepypioneer commented:
Hi @nerdyvaishali we have made those changes :)

v-thakkar committed:
Add space and update the date

Commit:
Change date on Team Popcorn's blog pos

Additionally, I changed the name of the pictures so that the img folder is more organized.
